src/Reflection/Annotations/AnnotationsPropertiesClassReflectionExtension.php

```diff
@@ -81,13 +81,11 @@ private function findClassReflectionWithProperty(
 		}
 
 		$parentClass = $classReflection->getParentClass();
-		while ($parentClass !== null) {
+		if ($parentClass !== null) {
 			$methodWithDeclaringClass = $this->findClassReflectionWithProperty($parentClass, $parentClass, $propertyName);
```

Comment:
this recursive call will already work its way thru the top parent class. we don't need another loop thru all parents